cli/cmd/turbo/main.go

```diff
@@ -12,13 +12,28 @@ import (
 	prune "turbo/internal/prune"
 	"turbo/internal/run"
 	uiPkg "turbo/internal/ui"
+	update "turbo/internal/update"
 	"turbo/internal/util"
 
 	"github.com/fatih/color"
 	"github.com/mitchellh/cli"
 )
 
 func main() {
+	// check for updates before running turbo
+	newRelease, _ := update.CheckforUpdate(
+		turboVersion,
+		"./update-check-history.yaml",
+		"https://api.github.com/repos/vercel/turborepo/releases/latest",
+	)
+	if newRelease != nil {
+		color.Yellow("A new version of `create-turbo` is available! %s")
+		fmt.Println("You can update by running: ")
+		// TODO: show only one method depending on the user chosen package manager
+		color.Cyan("yarn global add create-turbo Or npm i -g create-turbo")
```

Comment:
We can move this message to after config is loaded so that we can give correct installation instructions based on the detected pacakge manager. 

We also should not suggest global installation at the moment.